Fortnox.NET/WebSockets/Models/WebSocketCommandResponse.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace FortnoxNET.WebSockets.Models
+{
+    public class WebSocketCommandResponse
+    {
+        [JsonProperty(PropertyName = "response")]
+        public string Response { get; set; }
+
+        [JsonProperty(PropertyName = "result")]
+        public string Result { get; set; }
+
+        [JsonProperty(PropertyName = "tenantIds")]
+        public dynamic TenantIDs { get; set; } // TODO(Oskar): Don't use dynamic.
```

Comment:
@brokenprogrammer do you want to fix this before merge ?

Reply:
Its not necessary for a release but i think it should be an easy fix.
@BaconFinger the response we get back here is of the format:
```JSON
“tenantIds”: 
{
    “ouhohohhho89796”:34231,
    “lijldifdf856587”:29302
}

```

Reply:
so the properties can have arbitrary names? Seems strange.

Reply:
Yeah the arbitrary names are basically the accessToken and the value is the given ID for them.